src/ashley/defaults.py

```diff
@@ -36,3 +41,22 @@
         "instructor": _FORUM_ADMIN_PERMISSIONS,
     },
 )
+
+
+DRAFT_EXPORTER_ENTITY_DECORATORS = getattr(
```

Comment:
Much more explicit with 2 more letters? and coherent with your other variables names like `DRAFTJS_EXPORTER` below:
```suggestion
DRAFTJS_EXPORTER_ENTITY_DECORATORS = getattr(
```